Resources/Styles/ControlStyles.xaml

```diff
@@ -629,4 +629,133 @@
+    </Style>
+
+    <Style x:Key="ChequeViewButtonStyle" TargetType="{x:Type Button}">
+        <Setter Property="Width" Value="110" />
+        <Setter Property="Height" Value="50" />
+        <Setter Property="HorizontalAlignment" Value="Center" />
+        <Setter Property="Margin" Value="5" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type Button}">
+                    <Border
+                Background="{StaticResource Tertiary}"
+                BorderBrush="{StaticResource Primary}"
+                BorderThickness="2"
+                CornerRadius="5">
+                        <ContentPresenter
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center"
+                    TextElement.FontFamily="Sora"
+                    TextElement.FontSize="15"
+                    TextElement.FontWeight="SemiBold"
+                    TextElement.Foreground="GhostWhite" />
+                    </Border>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewTitleStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="FontFamily" Value="Sora" />
+        <Setter Property="Foreground" Value="Black" />
+        <Setter Property="FontSize" Value="20" />
+        <Setter Property="FontWeight" Value="Bold" />
+        <Setter Property="Margin" Value="5" />
+        <Setter Property="HorizontalAlignment" Value="Center" />
+        <Setter Property="VerticalAlignment" Value="Center" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewContentStyle" TargetType="{x:Type StackPanel}">
+        <Setter Property="MinHeight" Value="210" />
+        <Setter Property="MinWidth" Value="420" />
+        <Setter Property="MaxHeight" Value="400" />
+        <Setter Property="MaxWidth" Value="800" />
+        <Setter Property="Margin" Value="5" />
+        <Setter Property="Background" Value="White" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewCanvasLineStyle" TargetType="{x:Type Line}">
+        <Setter Property="Stroke" Value="Black" />
+        <Setter Property="StrokeThickness" Value="0.6" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewCanvasRectangleStyle" TargetType="{x:Type Rectangle}">
+        <Setter Property="Width" Value="140" />
+        <Setter Property="Height" Value="35" />
+        <Setter Property="Fill" Value="White" />
+        <Setter Property="Stroke" Value="Black" />
+        <Setter Property="StrokeThickness" Value="2" />
+        <Setter Property="Canvas.Left" Value="260" />
+        <Setter Property="Canvas.Top" Value="76" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+        <Setter Property="VerticalAlignment" Value="Top" />
+    </Style>
+
+    <Style x:Key="ChequeViewPreviewPayeeStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="Grid.Row" Value="1" />
```

Comment:
The `Grid.Row`, `Grid.Column` and `Grid.ColumnSpan` is only available to use when there is a Grid as parent control. This violates the code reuse if this TextBlock is not in a Grid layout control.

Try to define the row, column and the related "span" on View is suitable already.